### PR TITLE
[Data] Add Skywalking label data

### LINKED ISSUE #714: Skywalking project data should be added for analysis.
I want to add some labeled data into OpenDigger to help us for our community analysis.

Label: Skywalking

Type: Project

Repos:

- apache/skywalking
- apache/skywalking-nodejs
- apache/skywalking-nginx-lua
- apache/skywalking-java
- apache/skywalking-website
- apache/skywalking-rocketbot-ui
- apache/skywalking-banyandb
- apache/skywalking-kubernetes
- apache/skywalking-docker
- apache/skywalking-infra-e2e
- apache/skywalking-query-protocol
- apache/skywalking-python
- apache/skywalking-client-js
- apache/skywalking-swck
- apache/skywalking-agent-test-tool
- apache/skywalking-data-collect-protocol
- apache/skywalking-eyes
- apache/skywalking-satellite
- apache/skywalking-cli
- apache/skywalking-showcase
- apache/skywalking-banyandb-java-client
- apache/skywalking-rust
- apache/skywalking-goapi
- apache/skywalking-kong
- apache/skywalking-kubernetes-event-exporter
- apache/skywalking-booster-ui
- apache/skywalking-ui
- apache/skywalking-oal-tool

Orgs/Users:

- SkyAPM

### PR DESCRIPTION
**Pull request by frank-zsy**
Signed-off-by: Frankzhaopku <syzhao1988@126.com>

Add `Skywalking` labeled data and close #714 .

As we use `repo_id` to identify a repo so we can not use regex to label data right now and all repos need to be listed in the config file.

There are 28 repos with `skywalking` prefix under `apache` org now and more repo need to be added manually in the future and actually we can develop some scripts to do this automatically.

### CODE REVIEW

**Comment by frank-zsy** [comment]
@wu-sheng I added `Skywalking` label to the data and please check if this makes sense for you. BTW, the project label `Skywalking` will be also labeled as Chinese project as well.

**Comment by frank-zsy** [comment]
@wu-sheng I added `Skywalking` label to the data and please check if this makes sense for you. BTW, the project label `Skywalking` will be also labeled as Chinese project as well.

File: labeled_data/projects/skywalking.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Skywalking
```
Comment (wu-sheng) [inline labeled_data/projects/skywalking.yml:1]: Is this case sensitive? SkyWalking is usually either `SkyWalking` or `skywalking`.

Comment (frank-zsy) [reply labeled_data/projects/skywalking.yml:1]: Got it.

**Comment by frank-zsy** [comment]
I have changed `Skywalking` into `SkyWalking` @wu-sheng .

**Comment by frank-zsy** [comment]
I have changed `Skywalking` into `SkyWalking` @wu-sheng .

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
Thank you for setting this up.

**Comment by frank-zsy** [comment]
My pleasure ~

**Pull request merged**